db/migrate/20180611172637_add_average_deploy_time_to_stages.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class AddAverageDeployTimeToStages < ActiveRecord::Migration[5.2]
+  def up
+    ActiveRecord::Base.transaction do
+      add_column :stages, :average_deploy_time, :float
```

Comment:
maybe use an integer that means seconds then we don't need float math ?

Reply:
We need to use float math if we're keeping a running average, otherwise the formula above would not work